### ISSUE TITLE
halo state is incorrect when more than 2 terms are involved

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/equality-explorer#109]
From https://github.com/phetsims/equality-explorer/issues/103#issuecomment-388499584, as reported by @phet-steele in Slack:

> multitouch
hold a term over another to get a halo (do the Operations screen)
hold this term very still
use a different finger to add a term to the same one you are hovering over
the term you are holding will have its halo, but the term it was hovering over will not have a halo anymore

What's happening here is that two dragged terms don't know about each other, and are simultaneously controlling the halo state of the same term on the scale.   Definitely specific to multitouch.

**Comment by pixelzoom** [comment]
Something similar to this probably happens on the Numbers and Variables screen.  I have not confirmed yet, but the steps would be something like:

1. drag a term over an inverse term on the scale to get a halo, do not release, do not move
2. with a different finger, drag another term over the same inverse term on the scale, do not release, do not move
3. pull one of the terms away with either finger

There will still be 2 overlapping terms that sum to zero, but there will be no halo on the inverse term.  And again, this is specific to multitouch.

**Comment by pixelzoom** [comment]
I don't think this needs to be fixed for 1.0 release. And it may be something that we don't fix at all.  It's something that the user is highly unlikely to do: it involves multitouch, putting 2 terms so close together is physically challenging, the halo will be correct as soon a term is moved, and this doesn't result in any type of failure or incorrect result.  That said, I'll put some time into investigating.

**Comment by pixelzoom** [comment]
First place to start investigating is line 108 in TermDragListener:

`this.plate.contentsChangedEmitter.addListener( refreshHalosBound );`

I expected that this emitter/listener combination would handle these scenarios.

**Comment by pixelzoom** [comment]
The scenario I hypothesizes about in https://github.com/phetsims/equality-explorer/issues/109#issuecomment-392934757 is physically impossible for me to produce.  I can't drag two terms over the same inverse term on the scale.  Can't do it with 2 finger, or with 2 tablet pens.  So the probability that this will happen is even lower than what @phet-steele  reported, unlikely to ever happen in practice.

**Comment by pixelzoom** [comment]
I did reproduce the scenario originally reported in https://github.com/phetsims/equality-explorer/issues/109#issue-325856788.  

This is unlikely to ever happen, and if it does happen it's unlikely to cause any confusion. So resolving this issue has a benefit approaching zero.  It will be so costly to address, and I've already spent a couple of hours on it.  So I'm recommending that we do not address this, and live with it as a "known multitouch issue".  

@amanda-phet are you OK with not addressing this?
